cmd/kubeadm/app/master/manifests.go

```diff
@@ -370,25 +356,19 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted bool) [
 	return command
 }
 
-func getControllerManagerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted bool) []string {
-	var command []string
-
-	// self-hosted controller-manager needs to wait on a lock
-	if selfHosted {
-		command = []string{"/usr/bin/flock", "--exclusive", "--timeout=30", "/var/lock/controller-manager.lock"}
```

Comment:
Actually, there is an issue where flock was never used. This is why testing manifests is also important, so my bad here. Good catch, @luxas.